sdk/service.go

```diff
@@ -2,9 +2,20 @@ package sdk
 
 import (
 	"fmt"
+	"os"
 	"strings"
 )
 
 func FormatServiceName(owner, functionName string) string {
 	return fmt.Sprintf("%s-%s", strings.ToLower(owner), functionName)
 }
+
+func FormatGateway(URL string) string {
```

Comment:
This wouldn't be format `gateway` - it would be `createServiceURL()`. We should also allow the suffix to be passed in.

Reply:
Yeah ok I will reformat that

Reply:
I also think we may be better returning `url.URL` than string https://golang.org/pkg/net/url/#URL

Reply:
My first approach was with url.URL but I think its unnecessary complication. Just added simple if to resolve the port problem will amend now and check your feedback